packages/cli/src/bin/pipcook-plugin.ts

```diff
@@ -6,6 +6,9 @@ import ora from 'ora';
 import { listen, get } from '../request';
 import { route } from '../router';
 import { tunaMirrorURI } from '../config';
+import * as url from 'url';
+import { existsSync } from 'fs';
+import { ChildProcess } from 'child_process';
```

Comment:
The core modules should be moved in front of relative imports.

Reply:
Done.